apps/emqx/src/emqx_secret_loader.erl

```diff
@@ -22,9 +22,13 @@
 
 -export_type([source/0]).
 
--type source() :: {file, file:filename_all()}.
+-type source() :: {file, string() | binary()}.
 
 -spec load(source()) -> binary() | no_return().
+load({file, <<"file://", Path/binary>>}) ->
+    file(Path);
+load({file, "file://" ++ Path}) ->
```

Comment:
Nit. RIght now it's not possible to wrap a `string()`, only `binary()`.

Reply:
right. but I'd keep this clause here just in case.